features.json

```diff
@@ -95,13 +95,14 @@
 				"bulkMemory": "75",
 				"exceptions": "95",
 				"extendedConst": ["flag", "Requires flag `--js-flags=--experimental-wasm-extended-const`"],
+				"memory64": ["flag", "Requires flag `--js-flags=--experimental-wasm-memory64`"],
 				"multiValue": "85",
 				"mutableGlobals": "74",
 				"referenceTypes": "96",
 				"saturatedFloatToInt": "75",
 				"signExtensions": "74",
 				"simd": "91",
-				"tailCall": ["flag", "Requires flag `#enable-experimental-webassembly-features`"],
+				"tailCall": ["flag", "Requires flag `#enable-experimental-webassembly-features` in `about://flags`"],
```

Comment:
Actually, isn't it chrome://flags? I'm not sure about://flags even works in Chrome.

Reply:
Also, since they're linkable anyway, I wonder if we should just use the link here like `chrome://flags/#enable-experimental-webassembly-features`.

Sure, it can't be opened directly for security reasons, but if copy-pasted into the address bar, it will point to the right setting immediately.

Reply:
`about:flags` and `about://flags` both redirect to `chrome://flags`. I thought about:// might be more familiar to developers as other browsers use `about:` URLs too. But I could go either way on that.

Reply:
> I thought about:// might be more familiar to developers as other browsers use `about:` URLs too.

Ah interesting, I guess it does depend on specific browser / Chromium build then. I'm fine with `about:` too then.

Reply:
Yeah that's a good point. If we wanted to be extra clear we could even change the other flag references to be something like "Requires command-line flag `--experimental-wasm-foo`"

Reply:
I think "Requires flag --something" should be already clear to most developers that it's a command-line flag.